src/main/java/teammates/storage/entity/FeedbackQuestion.java

```diff
@@ -85,6 +97,24 @@ public FeedbackQuestion(
         this.showResponsesTo = showResponsesTo;
         this.showGiverNameTo = showGiverNameTo;
         this.showRecipientNameTo = showRecipientNameTo;
+        this.setCreatedAt(new Date());
+    }
+    
+    public Date getCreatedAt() {
+        return this.createdAt;
+    }
+    
+    public Date getUpdatedAt() {
+        return this.updatedAt;
+    }
+    
+    public void setCreatedAt(Date newDate) {
+        this.createdAt = newDate;
+        this.updatedAt = newDate;
+    }
+    
+    public void setLastUpdate(Date newDate) {
+        this.updatedAt = (keepUpdateTimestamp) ? this.updatedAt : newDate;
```

Comment:
While one-liners are nice and all, an `if`` may be clearer here, modifying the date only when `keepUpdateTimestamp` is not true (and doing nothing otherwise).